spec/lib/secure_headers/configuration_spec.rb

```diff
@@ -29,16 +29,14 @@ module SecureHeaders
       expect_default_values(header_hash)
     end
 
-    it "copies all config values except for the cached headers when dup" do
+    it "copies config values when duping" do
```

Comment:
Would it also make sense to add a test to verify that overrides never end up touching `csp` and only touch `dynamic_csp`?